wrapanapi/systems/redfish.py

```diff
@@ -31,7 +28,53 @@ def __init__(self, system, raw=None, **kwargs):
         if not self._odata_id:
             raise ValueError("missing required kwargs: 'odata_id'")
 
-        super(RedfishServer, self).__init__(system, raw, **kwargs)
+        super(RedfishResource, self).__init__(system, raw, **kwargs)
+
+    def _identifying_attrs(self):
+        """
+        Return the list of attributes that make this instance uniquely identifiable.
+
+        These attributes identify the instance without needing to query the API
+        for updated data.
+        """
+        return {'ems_ref': self.name}
```

Comment:
The problem is that Redfish doesn't have an uuid.

Reply:
@matejart but I can see the `def uuid` property implemented in L66 ;)

Reply:
This has been marked as resolved but @miha-plesko and @matejart you don't seem to be on the same page here. There is a UUID property defined, using the raw `Id` property. Why not use it here? If there is a valid reason, please include in-line in a comment why name is used.

Reply:
I am a bit unsure about what is the recommended practice in this context. For Redfish resource, the property `name` returns a unique idenfier of the instance. Considering this, would it make sense to return simply `{'name': self.name}`?
So unless `def uuid` is required, I would remove it because it is confusing and because a Redfish resource instance does not (necessarily) have a UUID.

But if I have to implement `def uuid`, then I will change as suggested to `{'uuid': self.uuid}` or `{'ems_ref': self.uuid}`.